concordia/tasks.py

```diff
@@ -89,9 +90,84 @@ def site_report():
     for campaign in Campaign.objects.all():
         campaign_report(campaign)
 
+    for topic in Topic.objects.all():
+        topic_report(topic)
 
-def campaign_report(campaign):
 
+def topic_report(topic):
+    report = {
+        "assets_not_started": 0,
+        "assets_in_progress": 0,
+        "assets_submitted": 0,
+        "assets_completed": 0,
+    }
+
+    asset_count_qs = (
+        Asset.objects.filter(item__project__topics=topic)
+        .values_list("transcription_status")
+        .annotate(Count("transcription_status"))
+    )
+
+    for status, count in asset_count_qs:
+        logger.debug("Topic %s assets %s: %d", topic.slug, status, count)
+        report[f"assets_{status}"] = count
+
+    assets_total = Asset.objects.filter(item__project__topics=topic).count()
+    assets_published = (
+        Asset.objects.published().filter(item__project__topics=topic).count()
+    )
+    assets_unpublished = (
+        Asset.objects.unpublished().filter(item__project__topics=topic).count()
+    )
+
+    items_published = Item.objects.published().filter(project__topics=topic).count()
+    items_unpublished = Item.objects.published().filter(project__topics=topic).count()
```

Comment:
The `Item.objects.published()` start here looks like it should be `unpublished`?

```suggestion
    items_unpublished = Item.objects.unpublished().filter(project__topics=topic).count()
```